bcda/models/service.go

```diff
@@ -261,23 +266,37 @@ func (s *service) getNewAndExistingBeneficiaries(ctx context.Context, conditions
 
 	var (
 		cutoffTime time.Time
+		upperBound time.Time
 	)
 
-	if s.stdCutoffDuration > 0 {
+	// if the _since parameter is after the attribution date,
+	// we should only return "old" benes; set the upper bound
+	// to the attribution date to avoid returning benes
+	// that were assigned after the attribution date
+	if !conditions.attributionDate.IsZero() &&
```

Comment:
Since we make the check [here](https://github.com/CMSgov/bcda-app/pull/644/files#diff-58fa56dc44c74157c85532e295a887dd487681523c03de7b7161263f856312e4R128), we don't need this in here right? We should assume the since can be satisfied and we can query for new/old benes?

Reply:
the `hasAttributionDate` check isn't run in the `getNewAndExisitingBeneficiaries` block (since that could be a valid path if the ACO isn't terminated - and we set the attr details to time.Time{} both if an ACO uses latest attribution OR if an ACO is not terminated)

Reply:
cleared up per offline conversation, we can remove this whole first if condition :)